lib/src/cli/deployapps/deployapps_command.dart

```diff
@@ -67,9 +67,13 @@ class DeployAppsCommand extends Command<void> {
     final apps = await client.getOrCreateApps();
 
     print('App import is complete. There are: ${apps.length} apps on the server:');
+    String listApps = '';
```

Comment:
Why not make that an actual list and then use `string.join` when printing the ids?

Reply:
Done, even though I don't need these appIds anymore.

Reply:
if its not needed lets remove it, but I think dumping the existing apps could help to debug something in the future.